tests/rest/test_rest_index.py

```diff
@@ -66,10 +66,11 @@ def test_route_endpoints(self):
 
             json_resp.sort()
 
-            assert json_resp[1]['url'].endswith(":9090/index.html#/dashboard/elasticsearch/Example SSH Dashboard") is True
+            assert json_resp[1]['url'].endswith("/app/kibana#/dashboard/dashboard/Example SSH Dashboard") is True
```

Comment:
Good find, this was due to the rest config, I fixed that and the tests in https://github.com/mozilla/MozDef/pull/402/commits/a695692382f34cd60e707d2ddf7f871998aebcdc and https://github.com/mozilla/MozDef/pull/402/commits/bf465b0480138c5bb65eaa159ffd2d686bcfb0d8